src/app/entity-groups/journal-entities/item-list-elements/sidebar-search-list-elements/journal/journal-sidebar-search-list-element.component.spec.ts

```diff
@@ -35,6 +38,11 @@ const parent = Object.assign(new Collection(), {
   },
 });
 
-describe('JournalSidebarSearchListElementComponent',
-  createSidebarSearchListElementTests(JournalSidebarSearchListElementComponent, object, parent, 'parent title', 'title', '1234, 5678'),
-);
+const expectedHierarchicalTitle = getExpectedHierarchicalTitle(parent, object);
+if (expectedHierarchicalTitle) {
```

Comment:
This `if` clause in **all tests** appears to be **skipping the test**.  When I run these tests locally (or look at the logs of the tests in GitHub), I cannot find any proof of the "JournalSidebarSearchListElementComponent" test being run below.

Are you sure these tests are running?  Could we remove this `if` statement to prove they work?